clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonReservationReportBuilder.groovy

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.aws.model
+
+import com.netflix.spinnaker.clouddriver.aws.model.AmazonReservationReport.OverallReservationDetail
+
+import java.util.concurrent.atomic.AtomicInteger
+
+interface AmazonReservationReportBuilder {
+  AmazonReservationReport build(AmazonReservationReport source)
+
+  /**
+   * This version (v3) of the reservation report:
+   *
+   * - normalizes all regional reservations to `xlarge` instance types
+   * - attempts to cover all shortfalls with regional reservations (converting `xlarge` to the necessary instance types)
+   */
+  class V3 implements AmazonReservationReportBuilder {
+    // relative multipliers for instance types < xlarge where the baseline is xlarge
+    // (would be nice if AWS had an API to derive this from)
+    Map<String, Double> instanceTypeMultipliers = [
```

Comment:
Pulled this from the AWS EC2 Instances site ... worth double checking for accuracy. 

I'm not aware of another source for this information (other than perhaps https://aws.amazon.com/blogs/aws/new-aws-price-list-api/).